multicore_runtime/wsgi_test.py

```diff
@@ -21,36 +20,50 @@
 import threading
 import unittest
 
+from . import wsgi_config
+
 from mock import MagicMock
 from mock import patch
-from werkzeug.test import Client
-from werkzeug.wrappers import Request
-from werkzeug.wrappers import Response
+from werkzeug import test
+from werkzeug import wrappers
 
 from google.appengine.api import appinfo
 
+def script_path(script, test_name=__name__):
+  """Returns a fully qualified module path based on test_name."""
+  return '{test_path}.{script}'.format(
+      test_path=test_name, script=script)
+
+def static_path(relative_path, test_path=os.path.dirname(__file__)):
+  """Returns a fully qualified static file path based on test_path."""
+  return '{base}{slash}{relative_path}'.format(
```

Comment:
Sorry I missed this before... but how is this different from `os.path.join(test_path, relative_path)`?